packages/synthetix-main/contracts/modules/account/AccountTokenModule.sol

```diff
@@ -0,0 +1,6 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-modules/contracts/modules/NftModule.sol";
+
+contract AccountTokenModule is NftModule {}
```

Comment:
Please, apply this change taking into account @sunnyvempati 's PR to change the Account NFT as ERC721_Enumerable